vsphere/tests/common.py

```diff
@@ -493,14 +493,24 @@ def build_rest_api_client(config, logger):
 NETS_INVALID.append(net_invalid)
 
 # VM invalid ip stack
-ip_route_invalid = vim.net.IpRouteConfigInfo.IpRoute()
-ip_route_invalid.prefixLength = 32
-ip_route_invalid.network = 'fe83::'
 ip_stack_invalid = vim.vm.GuestInfo.StackInfo()
 ip_stack_invalid.dnsConfig = None
 IP_STACKS_INVALID = vim.ArrayOfAnyType()
 IP_STACKS_INVALID.append(ip_stack_invalid)
 
+# VM invalid ip route gateway
+ip_route_invalid2 = vim.net.IpRouteConfigInfo.IpRoute()
```

Comment:
I ask out of curiosity: Why ip_route_invalid2? Do we have a ip_route_invalid or ip_route_invalid1?